src/server/graphql/mutations/updateReflectionIsEditing.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Changes the editing state of a retrospective reflection.
+ *
+ * @flow
+ */
+import type {AuthToken} from 'universal/types/auth';
+
+import {GraphQLID, GraphQLNonNull, GraphQLBoolean} from 'graphql';
+
+import getRethink from 'server/database/rethinkDriver';
+import UpdateRetroReflectionIsEditingPayload from 'server/graphql/types/UpdateReflectionIsEditingPayload';
+import {isTeamMember} from 'server/utils/authorization';
+import RethinkDataLoader from 'server/utils/RethinkDataLoader';
+import publish from 'server/utils/publish';
+import {TEAM} from 'universal/utils/constants';
+
+type Args = {
+  isEditing: boolean,
+  reflectionId: string,
+};
+
+type Context = {
```

Comment:
lets import this guy, every query, mutation, and subscription should have the same context.